variables.tf

```diff
@@ -148,25 +149,25 @@ variable "upgrade_operator" {
 
 # operator notification
 
-variable "notification_enabled" {
+variable "operator_notification" {
```

Comment:
the variable name has to be operator_notification_enabled

Reply:
no we are renaming, remember?

Reply:
in the doc and changelog its mentioned its getting changed to operator_notification_enabled.